### DISCUSSION TITLE
Rank Benefit Ideas

### DISCUSSION THREAD

**Discussion by Nathanwoodburn** [RunDavidMC/IoniFaucet#2] [Ideas]
Hi,
Please share your ideas for rank benefits.
We've already got:
0. No Benefits
1. Namebase referral link

We need 3-15 (I think)

**Comment by Nathanwoodburn** [comment]
Maybe for 15, we could allow a custom vanity URL

**Reply by RunDavidMC** [reply]
That could work. For them to get to that rank, would have have to have, say, 30-50 referrals to IoniFaucet?

**Comment by RunDavidMC** [comment]
Rank: Benefits - How to obtain
0: Can login and view the dashboard - Creating an account
3: Gets a referral code - Verifying email
10: Referred users are presented with referrer's NB referral code - 10-25 referrals to IoniFaucet
15: Gets a custom vanity URL - 35-50 referrals to IoniFaucet
25: Gets access to User Pages - 150 referrals to IoniFaucet, a major contribution of ideas or bug reporting, or some fee of HNS
35: Gets access to logistics, such as people's quiz answers, total names left, etc. - Would be mainly for people like Johnny Wu or developers
75: Administrator

**Reply by RunDavidMC** [reply]
What do you think?

**Reply by Nathanwoodburn** [reply]
Yeah sounds good

**Reply by Kano0403** [reply]
Maybe change 75: to Developer instead of Administrator? I just feel like it goes better for what it means.

**Reply by tiMaxal** [reply]
dev yeh .. admin doesn't hav to'v been dev .. is 35 an 'admin'?

**Reply by tiMaxal** [reply]
maybe 'contribution greater than' in lieu of 'fee'
